docs/gen-ai/README.md

```diff
@@ -0,0 +1,20 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: AI
```

Comment:
Given that you've felt the need to qualify the folder names with "gen-" rather than just use "ai", for consistency, I suspect that it makes more sense to use the following `linkTitle`, WDYT?

```suggestion
linkTitle: Generative AI
```

Reply:
I didn't want to claim the entire AI space. We're just forcing on LLM and maybe gen AI for now. If you think that's ok to "claim" this as "AI" then I can change it.